README.md

```diff
@@ -66,6 +66,14 @@ Following are the implementations managed by third-parties adopting the standard
   * Machine API Operator, https://github.com/openshift/machine-api-operator/tree/master
   * Machine-controller-manager, https://github.com/gardener/machine-controller-manager/tree/cluster-api
 
+## Versioning, Maintenance, and Compatibility
+
+- We follow [Semantic Versioning (semver)](https://semver.org/).
+- Cluster API release cadence is Kubernetes Release + 6 weeks.
+- The cadence is subject to change if necessary, refer to the [Milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) page for up-to-date information.
+- The _master_ branch is where development happens, this might include breaking changes.
+- The _release-X_ branches contain stable, backward compatible code. Every major (X) release, a new such branch is created.
```

Comment:
```suggestion
- The _release-X_ branches contain stable, backward compatible code. A new _release-X_ branch is created at every major (X) release.
```
Rewording because I read the current version twice before I was sure I understood the intended meaning.

Reply:
Fixed, thanks!